shuup/admin/forms/_auth.py

```diff
@@ -48,6 +49,29 @@ def clean_username(self):
 
         return username
 
+    def clean(self):
+        username = self.cleaned_data.get("username")
+        password = self.cleaned_data.get("password")
+
+        if username is not None and password:
+            self.user_cache = authenticate(
+                self.request, username=username, password=password
```

Comment:
`authenticate(request=self.request, username=username, password=password)` likely fixes it for Django 1.8 and 1.9. Applies to front too.